enhancements/monitoring/alerts-ui-management.md

```diff
@@ -0,0 +1,631 @@
+
+### User Stories
+
+1. **Bulk disable that are not required**
+   As a cluster administrator, I want to select and disable multiple alerts in one action,
+   so that I can permanently suppress non‑critical notifications and reduce noise.
+
+2. **Disable a built-in alerting rule**
+   As a cluster administrator, I want to disable an out-of-the-box alerting rule permanently, because it's not relevant for my environment (for instance, I don't want to see any alert about resource over-commitment because my cluster is only used for testing).
+
+3. **Replace an existing platform alerting rule by a custom definition.**
+   As a cluster admin, I want to disable existing built‑in alert, clone it and adjust its threshold and severity, and save it as a user‑defined rule, so that I can tailor default monitoring to suit my team’s SLAs without modifying upstream operators.
+
+4. **Create a custom alerting rule**
+   As a cluster admin/developer, I want to create an alerting rule using a form which allows me to specify mandatory fields (PromQL expression, name) and recommended fields (for duration, well-known labels/annotations such as severity, summary).
+
+5. ***Clone an alert based on an existing alerting rule (platform or user-defined)**
+   As a cluster admin/developer, I want to clone an alert and update it based on my organization needs.
+
+7. **View silence status on active alerts** - Exists today. Keep Functionality as is.
+   As a cluster admin, I want the “Active Alerts” list to show which alerts are currently silenced (and until when),
+   so that I have clear visibility into which notifications are suppressed and why.
+
+8. **Create an alerting rule from the Metrics > Observe page** - Currently not in scope
+   AS a cluster admin I would like to be able to create an alert directly from the Metrics > Observe page,
+   after I used it to tune the expression that I need.
+
+## Goals
+1. Add a Console UI for managing alerts.
+2. Standardize `group` and `component` labels on alert rules to clearly surface priority and impact, to help administrators to understand what to address first.
+3. CRUD operations on user‑defined alerts via UI and API.
+4. Clone platform or user alerts to customize thresholds or scopes.
+5. Disable/enable alerts by creating/updating entries in the `AlertRelabelConfig` CR.
+6. Create/manage silences in Alertmanager and reflect this in the UI. - Already exists today.
+7. Aggregate view of all alerting rules, showing definitions plus relabel context.
+8. Aggregate view of all alerts, showing status (Pending, Firing, Silenced) and relabel context.
```

Comment:
Do these two points assume some kind of mapping between the alert/alerting rule and its corresponding relabel context? There's no explicit connection or contract right now if I am not mistaken.